### PR TITLE
New package: slmenu

### PR DESCRIPTION
**Pull request by benalb**

### CODE REVIEW

File: srcpkgs/slmenu/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'slmenu'
+pkgname=slmenu
+version=0.1
+revision=1
+_git_commit=7e74fa5db73e
+_author=rafaelgg
```
Comment (cr6git) [inline srcpkgs/slmenu/template]: No need to use a custom variable just for that.

Comment (benalb) [reply srcpkgs/slmenu/template]: I adapted the template from xss-lock, there is no version, nor releases in the repo.

Comment (cr6git) [reply srcpkgs/slmenu/template]: I mean this custom variable: `_author=rafaelgg`

Comment (benalb) [reply srcpkgs/slmenu/template]: get rid of _author and 
distfiles="${homepage}/get/${_git_commit}.tar.gz"   ?

Comment (cr6git) [reply srcpkgs/slmenu/template]:
````
wrksrc="rafaelgg-${pkgname}-${_git_commit}"
distfiles="https://bitbucket.org/rafaelgg/${pkgname}/get/${_git_commit}.tar.gz"
````

Comment (benalb) [reply srcpkgs/slmenu/template]: ok, got it. Thank you for your patience.

**Pull request merged**